server/database/database.js

```diff
@@ -0,0 +1,38 @@
+const mysql = require('mysql2');
+const dotenv = require('dotenv');
+dotenv.config();
+
+const pool = mysql.createPool({
+	host: process.env.MYSQL_HOST,
+	user: process.env.MYSQL_USER,
+	password: process.env.MYSQL_PASSWORD,
+	database: process.env.MYSQL_DATABASE
+}).promise();
+
+async function getLoginInfo(email) {
+    const query = "SELECT * FROM login_info WHERE email = ?";
+    const { rows } = await pool.query(query, [email]);
+    return rows;
+}
+
+async function getUser(username) {
+    const query = "SELECT * FROM user WHERE username = ?";
+    const { rows } = await pool.query(query, [username]);
```

Comment:
syntax here is wrong

Reply:
are you referring to the curly braces where I should have used brackets?